ocf_datapipes/batch/convert_xarray_batch_to_numpy_batch.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+import xarray as xr
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+from ocf_datapipes.utils.consts import NumpyBatch
+
+
+@functional_datapipe("convert_xarray_batch_to_numpy_batch")
+class ConvertXarrayBatchToNumpyBatch(IterDataPipe):
+    def __init__(self, source_dp: IterDataPipe):
+        self.source_dp = source_dp
+
+    def __iter__(self) -> NumpyBatch:
+        for xr_batch in self.source_dp:
+            # TODO Actually convert it, or add conversion to XarrayBatch
```

Comment:
do we need this todo still?

Reply:
No, the ones in `convert` now make the numpyBatch as in the PP training script now, so I'll get ride of these. I'll keep the merge_batchml to still be able to make the BatchML to save out in prod